### ISSUE TITLE
Live Reload is not possible without body or head tag

### ISSUE THREAD

**Issue by lukaszschab** [ritwickdey/vscode-live-server#158]
I'm submitting a problem --> I just install Live Server and I try to open a few html document (with head and body ;) ) --> but every time I want to see a Live Preview of this documents a get a blank page in Chrome and notification: "Live Reload is not possible without body or head tag." Do I need to configuration Live Server in some way or it is a bug or I doing something wrong?

**Comment by ritwickdey** [comment]
pls share the html page

**Comment by rjoydip** [comment]
@lukaszschab This is not a valid HTML. Please wrap your code with HTML. Right now live-server doesn't work on without `html` and `body`.

**Comment by lukaszschab** [comment]
Yeah :) I am new to github ;) I just fund that adding ``` is helpful :)

**Comment by lukaszschab** [comment]
Also, I created a new HTML page with code bellow and  Live Reload is working on that file :) but not on that above :( 

One difference that I see (I don't know if that's important or not is that above code is related with Git repository on my local computer and code bellow is't.

```
<head> 

</head>


<body>

Text

    
</body>

```

**Comment by ritwickdey** [comment]
Apart from the error message, What you're getting output in browser screen? Is this not valid output? or the live reload feature is not working?

**Comment by lukaszschab** [comment]
On VSC I get:

![image](https://user-images.githubusercontent.com/28892022/41500608-427e84a6-7195-11e8-86a4-d3673d661274.png)

On Chrome I get blank page:

![image](https://user-images.githubusercontent.com/28892022/41500615-6d0a0e5c-7195-11e8-99f9-e63e6613c764.png)

I shuld check something else?

**Comment by ritwickdey** [comment]
& in the browser console?

**Comment by lukaszschab** [comment]
Console is blank:

![image](https://user-images.githubusercontent.com/28892022/41500661-58f9aca0-7196-11e8-8289-f958c04365a4.png)

In Elements I see:

![image](https://user-images.githubusercontent.com/28892022/41500666-6a050de6-7196-11e8-8733-c6eefa5d9234.png)

Should I check something else?

**Comment by lukaszschab** [comment]
I try to open an HTML page which I have a problem with new windows for VSC and I open a folder with them. Then I try to Go Live i get:

"Error on port 5500. Please try to change the port through settings or report on GitHub."

![image](https://user-images.githubusercontent.com/28892022/41500752-0733c02a-7198-11e8-8432-503efff65e00.png)

**Comment by lukaszschab** [comment]
I change in the setting port to 0 and now it's working on new window :)

**Comment by lukaszschab** [comment]
It still wouldn't work on the previous window but I just close and open VSC again and know it working on port 5500 :)

**Comment by ritwickdey** [comment]
Because port 5500 is busy as you already started live server in another window.

Anyway, is the previous issue solved? I think you're checking wrong file in browser. 

What is the name of the file? And if you right-click on the file & click to the 'open with live server', what is happening? & please share a screenshot of your project directory that's opened in vscode?

**Comment by lukaszschab** [comment]
Now and then I was clicking inside a code in document --> right mouse button and select Open with Live Server :)

**Comment by lukaszschab** [comment]
Everything is all right now :) Thanks 👍  :)

**Comment by lukaszschab** [comment]
Done :)

**Comment by tzwtmll** [comment]
> Everything is all right now :) Thanks 👍 :)
i still havent solved it,pls share your function

**Comment by tzwtmll** [comment]
> Everything is all right now :) Thanks 👍 :)
i still havent solved it,pls share your function
